chalice/config.py

```diff
@@ -155,9 +155,9 @@ def lambda_python_version(self) -> str:
         # 3.11 and higher will use 3.11
```

Comment:
isn't this whole property needing to be removed? AWS supports only Python 3.8+ anyway...